Products/GenericSetup/tests/common.py

```diff
@@ -32,6 +32,12 @@ class DOMComparator:
 
     def _compareDOM(self, found_text, expected_text, debug=False):
 
+        if six.PY3:
+            if isinstance(found_text, bytes):
+                found_text = found_text.decode('utf8')
+            if isinstance(expected_text, bytes):
+                expected_text = expected_text.decode('utf8')
+
```

Comment:
Ironic that `xml.minidom.parseString` wants text, given that XML is *by definition* bytes, encoded as 'utf8' unless otherwise specified via header or BOM.